--- a/src/charsys.c
+++ b/src/charsys.c
@@ -148,7 +148,7 @@
  * [CHANGING THIS WILL CAUSE SECURITY/SYNCH PROBLEMS AND WILL
  *  VIOLATE YOUR ""RIGHT"" ON SUPPORT IMMEDIATELY]
  */
-const char *illegalnickchars = "\xA0!+%@&~#$:'\"?*,.";
+const char *illegalnickchars = "!+%@&~#$:'\"?*,.";
 
 /** Called on boot and just before config run */
 void charsys_reset(void)
--- a/src/channel.c
+++ b/src/channel.c
@@ -1213,7 +1213,7 @@
 		 * or some such.
 		 *   --Wizzu
 		 */
-		if (*ch < 33 || *ch == ',' || *ch == 160)
+		if (*ch < 33 || *ch == ',')
 		{
 			*ch = '\0';
 			return;
